debian/changelog

```diff
@@ -1,25 +1,24 @@
-cloud-init (25.1-0ubuntu1~20.04.1) UNRELEASED; urgency=medium
+cloud-init (25.1.1-0ubuntu0~20.04.1) focal; urgency=medium
 
   * Drop cpicks which are now upstream:
     - d/p/cpick-d75840be-fix-retry-AWS-hotplug-for-async-IMDS-5995
     - d/p/cpick-84806336-chore-Add-feature-flag-for-manual-network-waiting
-  * refresh patches
+  * refresh patches and sort hunks within all patches (--sort quilt refresh)
```

Comment:
These were independent refreshes as we thought it'd be best to call out what has changed independent of what was sorted. IIRC the list of patches we sorted will include every single patch so there was no need to call them out there.

Reply:
that works for me to keep it separate.